### ISSUE TITLE
Ability to test different visual breakpoints for responsive CSS?

### ISSUE THREAD

**Issue by benfrain** [jamesshore/quixote#8]
I'm just looking to set up CSS testing for an App I'm working on. I've been looking at a few options and I'm liking the look of Quixote. 

However, one thing I'm curious about. I'd like to test some visuals at certain breakpoints as it is a responsive app. So, in laymans terms, it would be great if I could effectively ask Quixote, 'at 600px plus, is .hdr-Logo 11px from .hdr-Nav' for example.

Can Quixote facilitate this? Couldn't find anything specific in the 

Thanks, Ben

**Comment by jamesshore** [comment]
Hi Ben, thanks for giving it a try. You should be able to use `frame.resize(width, height)` to do what you want. It's documented [here](https://github.com/jamesshore/quixote/blob/master/docs/QFrame.md#frameresize).

**Comment by benfrain** [comment]
Ah! Fantastic. Sorry I missed that.
